token.temp.py

```diff
@@ -1 +1,2 @@
-TOKEN = ''
\ No newline at end of file
+BOT_TOKEN = ''
+TEXT_TOKEN = ""
```

Comment:
We have `''` in one place and `""` in another, use one standard, lets move to `''`